### PR TITLE
rustup

### PR DESCRIPTION
**Pull request by bfops**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ build = "src/gl/build.rs"
 name = "gl"
 path = "src/gl/lib.rs"
 
-[build-dependencies.gl_generator]
+[dependencies.gl_generator]
```
Comment (tomaka) [inline Cargo.toml]: Why this change? This is wrong!

Comment (bfops) [inline Cargo.toml]: The tests wouldn't build otherwise. Should I add it under `build-dependencies` and `dev-dependencies`?

Comment (tomaka) [inline Cargo.toml]: Oh, that's because I forgot to remove `extern crate gl_generator;` from the tests.

File: tests/generators_symbols.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#![feature(phase)]
+#![feature(plugin)]
 
-#[phase(plugin)]
+#[plugin]
 extern crate gl_generator;
```
Comment (tomaka) [inline tests/generators_symbols.rs]: Please remove this entirely. It is not needed.

File: tests/no_warning.rs [hunk 3]
```diff
@@ -1,9 +1,9 @@
-#![feature(phase)]
+#![feature(plugin)]
 
 //! Making sure that no warning is generated by code from generate_gl_bindings!
 #![deny(warnings)]
 
-#[phase(plugin)]
+#[plugin]
 extern crate gl_generator;
```
Comment (tomaka) [inline tests/no_warning.rs]: And this too.

Comment (bfops) [inline tests/no_warning.rs]: Done!

**Comment by tomaka** [comment]
Thanks
